### PR TITLE
CDPS-713: added arrived today page

### PR DESCRIPTION
**Pull request by whitfield-mj**

### CODE REVIEW

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -35,6 +35,7 @@ generic-service:
     SOC_API_URL: https://manage-soc-cases-api-dev.hmpps.service.justice.gov.uk
     TOKEN_VERIFICATION_API_URL: https://token-verification-api-dev.prison.service.justice.gov.uk
     WHEREABOUTS_ENDPOINT_URL: https://whereabouts-api-dev.service.justice.gov.uk
+    PRISONER_SEARCH_API_URL: https://prisoner-search-dev.prison.service.justice.gov.uk
```
Comment (brightonsbox) [inline helm_deploy/values-dev.yaml:38]: Would be good to add this to: https://github.com/ministryofjustice/hmpps-digital-prison-services/blob/main/readme/building_and_running.md?plain=1#L93

File: integration_tests/e2e/arrivedToday.cy.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    page.arrivedTodayRows().first().find('td').eq(1).should('contain.text', 'Shannon, Eddie')
+    page.arrivedTodayRows().first().find('td').eq(2).should('contain.text', 'A1234AB')
+    page.arrivedTodayRows().first().find('td').eq(3).should('contain.text', '01/01/1980')
+    page.arrivedTodayRows().first().find('td').eq(4).should('contain.text', '1-1-1')
+    page.arrivedTodayRows().first().find('td').eq(5).should('contain.text', '10:30')
+    page.arrivedTodayRows().first().find('td').eq(6).should('contain.text', 'Cookham Wood')
+    page.arrivedTodayRows().first().find('td').eq(7).should('contain.text', '')
+  })
+
+  it('should display alerts and categor if cat A', () => {
```
Comment (brightonsbox) [inline integration_tests/e2e/arrivedToday.cy.ts:37]: meh.. typo of category

File: server/config.ts [hunk 3]
```diff
@@ -226,6 +234,7 @@ export default {
   },
   serviceUrls: {
     digitalPrisons: get('DIGITAL_PRISONS_URL', 'http://localhost:3001', requiredInProduction),
+    prisonerProfile: get('PRISONER_PROFILE_URL', 'http://localhost:3002', requiredInProduction),
```
Comment (brightonsbox) [inline server/config.ts:237]: Do we need this in environment config files, feature.env and readme file?

File: assets/js/govukFrontendInit.js [hunk 4]
```diff
@@ -1,3 +1,3 @@
-import { initAll } from '/assets/govuk/govuk-frontend.min.js'
```
[the remark this answers is not in the document]
Comment (whitfield-mj) [reply assets/js/govukFrontendInit.js:1]: whoops

File: server/services/utils/alertFlagLabels.ts [hunk 5]
```diff
@@ -0,0 +1,118 @@
+import { Alert, Prisoner } from '../../data/interfaces/prisoner'
+import { AlertFlagLabel } from '../interfaces/establishmentRollService/AlertTypeLabel'
+
```
Comment (brightonsbox) [inline server/services/utils/alertFlagLabels.ts:3]: With more prisoner specific stuff in this repo, we're starting to need to do some copy-paste of code from the profile.  So if we need to add an alert flag, it'll now need updating in two places.  One to keep an eye out for I suppose!

**Review by brightonsbox — APPROVED** [review]
LGTM

File: server/services/utils/alertFlagLabels.ts (hunk 5 above)
Comment (whitfield-mj) [reply server/services/utils/alertFlagLabels.ts:3]: Yeah that was also the case when this was in dps. I think we can look towards packaging stuff up when we get a bit of time and sharing common functionality. May be useful precedent to set in general for DPS

**Pull request merged**